.github/workflows/security.yml

```diff
@@ -0,0 +1,274 @@
+  #       run: sonar-scanner -Dsonar.projectKey=mac-fc-github-actions-runner-aws
+
+  #     - name: Run SAF CLI on SonarQube
+  #       uses: mitre/saf_action@v1
+  #       with:
+  #         command_string: "convert sonarqube2hdf -a ${{ secrets.SONAR_TOKEN }} -u ${{ secrets.SONAR_HOST_URL }} -n mac-fc-github-actions-runner-aws -o scans/sonarqube/sonarqube_hdf.json -b ${{ steps.get-branch-name.outputs.branch-name }}"
+
+  #     - name: Convert SonarQube HDF to ASFF
+  #       uses: mitre/saf_action@v1
+  #       with:
+  #         command_string: "convert hdf2asff -i scans/sonarqube/sonarqube_hdf.json -a 156322662943 -r us-east-1 -t sonarqube -o scans/sonarqube/asff"
+
+  #     - name: Upload SonarQube ASFF
+  #       uses: ./.github/actions/upload-asff
+  #       with:
+  #         asff-directory-path: scans/sonarqube/asff
+  #         company-name: SonarQube
+  #         product-name: SonarQube
+
+  # stop-runners:
+  #   name: Deprovision self-hosted runners
+  #   runs-on: ubuntu-latest
+  #   if: always()
+  #   needs: [start-runners, run-sonarqube]
+  #   steps:
+  #     - name: Configure AWS credentials
+  #       uses: aws-actions/configure-aws-credentials@v1
+  #       with:
+  #         aws-region: us-east-1
+  #         role-to-assume: ${{ secrets.OIDC_IAM_ROLE_ARN }}
+
+  #     - name: Scale down ECS service
+  #       uses: Enterprise-CMCS/ecs-scale-service@main
+  #       with:
+  #         cluster: gh-runner-6d81a626-1844-5df5-a3e3-cbbbeab84233
+  #         service: gh-runner-6d81a626-1844-5df5-a3e3-cbbbeab84233
+  #         desired-count: 0
+
+  slack-notification:
+    name: Slack notification
+    needs: [build-and-test-image, run-inspec] # set "needs" to all previous jobs to make sure this job runs last
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Check for Slack webhook secret
+        id: check-webhook-secret
+        run: echo '::set-output name=secret::${{ secrets.SLACK_WEBHOOK_URL }}'
+
+      - name: Get workflow conclusion # conclusion is stored in env.WORKFLOW_CONCLUSION
+        if: steps.check-webhook-secret.outputs.secret
+        uses: technote-space/workflow-conclusion-action@v3
+
+      - name: Send Slack notification
+        if: steps.check-webhook-secret.outputs.secret
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": "Security scan workflow #${{github.run_number}} ",
+                    "emoji": true
+                  }
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "${{ (env.WORKFLOW_CONCLUSION == 'success' && ':white_check_mark:') || ':x:' }} Result: ${{ env.WORKFLOW_CONCLUSION }}"
```

Comment:
This might be a nit, but should we consider a different emoji for cancelled vs. error? Are all cancellations manual, or would a job be automatically cancelled if it hit a timeout or some other limit? I'm just thinking about scanability when looking at the messages since `if: always()` is being used.

Reply:
Good point, I tried to trigger the different cases in the new emoji code but could only figure out how to do 'success', 'canceled', and 'failure'